index.md

```diff
@@ -1,18 +1,19 @@
 @def title = "Julia Berlin Users"
+---
 
 ## Upcoming events
 
-**10.02.2021** at **18:30** on [**Gather.Town**](https://gather.town/app/FKxW8cZ89KZSOjAm/berlinjuliausers) (you only need your browser)
+**10.02.2022** at **18:30** at the Weierstrass Institute (WIAS). There will be a maximum of 16 participants, with 2G rule (tested beforehand) and with masks.
```

Comment:
Das Datum ist in der Vergangenheit ;)

Reply:
Ups danke! I was like cool it's the same day :laughing: